src/modules/menu/components/menu/feedback/FeedbackToolbar.vue

```diff
@@ -5,25 +5,33 @@
             'me-2': showAsLinks,
         }"
     >
-        <FeedbackButton :show-as-link="showAsLinks" />
+        <FeedbackButton v-if="hasGiveFeedbackButton" :show-as-link="showAsLinks" />
+        <ReportProblemButton v-if="hasReportProblemButton" :show-as-link="showAsLinks" />
```

Comment:
Because this file is so small and you touch a lots of lines I would suggest to quickly change it to composition api